src/main/java/frc/team2412/robot/Controls.java

```diff
@@ -64,8 +65,8 @@ public Controls(Subsystems s) {
         codriverController = new XboxController(CODRIVER_CONTROLLER_PORT);
 
         resetDriveGyroButton = driveController.getRightJoystickButton();
-        shootButton = driveController.getRightBumperButton();
-
+        setPoseButton = driveController.getStartButton(); // set pose button is practice bot only
+        shootButton = driveController.getLeftBumperButton();
```

Comment:
Is there a reason the shoot button got moved to the other bumper?

Reply:
no 👍